### PR TITLE
CEDS-3134 Amendments to the 'TARIC additional code' page

### PR DESCRIPTION
**Pull request by udinobi**
Amendments, based on the commodity code specified or not, to the 'TARIC additional code' page

### CODE REVIEW

File: app/forms/declaration/TaricCodeFirst.scala [hunk 1]
```diff
@@ -35,12 +35,13 @@ object TaricCodeFirst extends DeclarationPage {
   implicit val format = Json.format[TaricCodeFirst]
 
   val hasTaricCodeKey = "hasTaric"
+  val none = TaricCodeFirst(None)
```
Comment (glidester) [inline app/forms/declaration/TaricCodeFirst.scala:38]: Nit: not sure what value this serves, looked cleaner with the literal value in the pattern match below as before

Comment (udinobi) [reply app/forms/declaration/TaricCodeFirst.scala:38]: It's a sort of static value and it's used in a few places, app and tests.

**Review by glidester — APPROVED** [review]
Looks good, just one minor comment.

Our implementations of basically the same thing (just on different pages) are very similar but they do differ slightly in execution in one or two places. Not a big deal and neither is 'better' than the other but it does mean there is now inconsistency in the service. Maybe a topic to cover in the next dev catchup! :grin:

File: app/forms/declaration/TaricCodeFirst.scala (hunk 1 above)
Comment (glidester) [reply app/forms/declaration/TaricCodeFirst.scala:38]: :+1:

**Pull request merged**